openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class is_usda_disabled(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Indicates whether person is disabled per USDA definition"
+    label = ""
+
+    def formula(persons, period, parameters):
+        disabled_status = parameters.usda.disabled.values
```

Comment:
Let's actually change `disabled.yaml` to `disabled_programs.yaml` then this will work:
```suggestion
        disabled_programs = parameters.usda.disabled_programs
```